README.md

```diff
@@ -1,6 +1,6 @@
 # ✨ TorchObjectDetectionSegmenter
 
-**TorchObjectDetectionSegmenter** is a class that ...
+**TorchObjectDetectionSegmenter** is a class that supports object detection and segmentation using PyTorch with Faster R-CNN and Mask R-CNN models.
```

Comment:
not segmentation, bounding box extraction or localization ...

Reply:
but this is a segmenter ? 

Read online that "Semantic Segmentation detects all the objects present in an image at the pixel level" and "Instance segmentation assigns a label to each pixel of the image."

Reply:
Yes, and this is the key. These 2 specific models do not do segmentation at pixel level. They propose bounding boxes.

Reply:
Mask R-CNN has an additional branch for predicting segmentation masks on each Region of Interest (RoI) in a pixel-to pixel manner ?